.github/workflows/ci.yml

```diff
@@ -32,4 +37,35 @@ jobs:
       shell: bash
     - name: Build
       run: make
-      
+
+  integration:
+    name: Integration testing
```

Comment:
I haven't read in detail what those scripts do, but it seems out-of-proportion to setup k8s, riff, etc just to test a glorified Kafka consumer. Setting up liiklus on some backend (even in-memory) should be sufficient IMO.

Reply:
> but it seems out-of-proportion to setup k8s, riff, etc just to test a glorified Kafka consumer

Actually, fats make it trivially easy to setup, and given that this is an integration test, I think we should exercise all of the moving parts.

We still need to invest in unit testing this.